hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/TestSnapshotScannerHDFSAclController.java

```diff
@@ -139,6 +140,13 @@ public static void tearDownAfterClass() throws Exception {
     TEST_UTIL.shutdownMiniCluster();
   }
 
+  private void snapshotAndWait(final String snapShotName, final TableName tableName)
+    throws Exception{
+    admin.snapshot(snapShotName, tableName);
+    LOG.info("Sleep for one second, waiting for HDFS Acl setup");
+    Threads.sleep(1000);
```

Comment:
Up this to 3 seconds? Yeah, this ugly (smile).

Reply:
I updated it to be 3 seconds, will wait for the test result and commit. The branch-2 results with patch looks good so far.